metadata/ComputerVision__5__metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "changeLog": "Release v2023.04",
+  "cpu": 0,
+  "description": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "displayName": "ComputerVision",
+  "gpu": 1,
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/user-guide/computer-vision ",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON36_CV",
+  "name": "ComputerVision",
+  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/user-guide/computer-vision ",
+  "processorType": "GPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Computer Vision",
+  "projectDescription": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "23.4",
+  "languageVersion": 3,
+  "version": 5,
+  "customVersion": "23.04.1",
+  "imagePath": "cv2304:23.4.5.2"
```

Comment:
Not clear if I should also have `parametersFileJSON` and `settings`... Guessing that those are not mandatory, right?

Reply:
Got it! We will configure probes in the next release.

Reply:
https://uipath.atlassian.net/browse/CV-621